src/main/java/nz/cri/gns/NZSHM22/opensha/enumTreeBranches/NZSHM22_DeformationModel.java

```diff
@@ -38,6 +38,26 @@
             "rupture set RmlsZTozMDMuMEJCOVVY",
             "slip_deficit_rates_no_prior_uniform-stddev_D90_RmlsZTozMDMuMEJCOVVY.dat"),
 
+    GEOD_NO_PRIOR_2010_RmlsZTo4NTkuMDM2Z2Rw_OFF_SWAP(
+            "geodetic, no geological prior constraint, 2010, offshore swap",
```

Comment:
use `geodetic, no geological prior constraint, 2010, offshore rates from geologic.`

Reply:
as that's what swap means apparently.